Compiling tutorial gives missing Kokkos header

If I copy step-1 and try to use an installed version of the current master (b206511199dd01f3a08f60a1ab850966c3d7fe87), I get a missing kokkos header:

```
[ 50%] Building CXX object CMakeFiles/step-1.dir/step-1.cc.o
In file included from /home/simon/bin/dealii/include/deal.II/base/numbers.h:28,
                 from /home/simon/bin/dealii/include/deal.II/base/config.h:579,
                 from /home/simon/bin/dealii/include/deal.II/grid/tria.h:20,
                 from /home/simon/workspace/step-1/step-1.cc:22:
/home/simon/bin/dealii/include/deal.II/bundled/Kokkos_Macros.hpp:65:10: fatal error: KokkosCore_config.h: No such file or directory
   65 | #include <KokkosCore_config.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/step-1.dir/build.make:76: CMakeFiles/step-1.dir/step-1.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:90: CMakeFiles/step-1.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

Looks like something is wrong with the cmake config related to kokkos? Ping @tamiko 

<!--
Thanks for reporting an issue with deal.II! This HTML comment won't be visible
in the issue tracker but is present here to help you write a better description
of the problem you encountered.

If you have questions about deal.II, please post them on the mailing list:

   https://groups.google.com/g/dealii

Otherwise:
- Do you encounter a compilation failure? Please attach the detailed.log file
  present in the build directory here. Any additional compiler messages will also
  be helpful!

- Does something crash (e.g., an example) that shouldn't? Please include any relevant
  terminal output as well as the detailed.log file generated in your build
  directory (should you have one).

- Is there a missing feature you want to implement? Please briefly describe it and
  the problem it solves.

- Did you find some other issue with deal.II? Summarize the problem here.
-->

--- a/bundled/CMakeLists.txt
+++ b/bundled/CMakeLists.txt
@@ -52,7 +52,9 @@ if(DEAL_II_FEATURE_KOKKOS_BUNDLED_CONFIGURED)
     ${KOKKOS_FOLDER}/tpls/desul/include/
     DESTINATION ${DEAL_II_INCLUDE_RELDIR}/deal.II/bundled
     COMPONENT library
-    FILES_MATCHING PATTERN "*.hpp"
+    FILES_MATCHING
+    PATTERN "*.hpp"
+    PATTERN "*.h"
     )
 endif()
 

